pkg/controller/capacitycontroller/capacitycontroller.go

```diff
@@ -130,17 +131,18 @@ func (d *Controller) reconcileDrive(ctx context.Context, drive *drivecrd.Drive)
 	case (health != apiV1.HealthGood && health != apiV1.HealthUnknown) ||
 		status != apiV1.DriveStatusOnline ||
 		usage != apiV1.DriveUsageInUse:
-		return d.handleInaccessibleDrive(ctx, drive.Spec)
+		return d.handleInaccessibleDrive(ctx, drive)
```

Comment:
do we also need this in LVG?

Reply:
no, lvg bad it's size will be reset to 0,  and the related lvg ac could not be used any more.